features/fixtures/ast.json

```diff
@@ -15,18 +15,20 @@
       "description": "<resource group description>\n\n",
       "resources": [
         {
+          "element": "resource",
           "name": "<resource name>",
           "description": "<resource description>\n\n",
           "uriTemplate": "/resource/{parameter}",
           "model": {
             "name": "<resource name>",
-            "description": "<resource model description>",
+            "description": "<resource model description>\n",
```

Comment:
Just making a small [fix](https://github.com/apiaryio/snowcrash/pull/281/files#diff-d3c5f2fc18729a968dbcb8059325dab9R17) in the blueprint file led to this.